core/modules/views/plugins/views_plugin_display_block.inc

```diff
@@ -85,7 +85,10 @@ class views_plugin_display_block extends views_plugin_display {
       $info = $this->get_option('block_description');
     }
     else {
-      $info = t('View: @view (@display)', array('@view' => t($this->view->get_human_name()), '@display' => t($this->display->display_title)));
+      $info = t('View: @view (display: @display)', array(
```

Comment:
love it!

Reply:
I didn't actually add the `View:` prefix here - I only added the `display:` prefix. The `View:` prefix was actually added already, which is actually causing problems with duplicated `View:` prefixes.

Because this is only added to certain views blocks (if there is no `block_description` added in the block display), you can end up with some views-provided blocks that have the prefix, and some that don't 😞 ...I had to add logic in `_layout_block_add_block_title_prefix()` to ensure that there aren't duplicated prefixes, while at the same time making sure that all views-provide blocks had the `Views:` prefix in the "Add block" dialog (so that they aren't "scattered" by the alphabetical sorting).

Reply:
...the reason why I added the `display:` prefix by the way is that we anded up with things like this (that is before any changes in this PR, and that is only one example that I randomly found):
`View: Layout Context Test (User account) (Block)`

In the above title, you don't know what the strings in the brackets are. Which one is the name of the display? My change adds the `display:` prefix, so you get this instead:
`View: Layout Context Test (User account) (display: Block)`